modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenModel.java

```diff
@@ -83,7 +92,7 @@ public class CodegenModel implements IJsonSchemaValidationProperties {
     public Set<String> allMandatory = new TreeSet<>(); // with parent's required properties
 
     public Set<String> imports = new TreeSet<>();
-    public boolean hasVars, emptyVars, hasMoreModels, hasEnums, isEnum, hasValidation;
+    public boolean hasVars, emptyVars, hasMoreModels, hasEnums, isEnum, hasValidation, hasNullable;
```

Comment:
might be you've missed to push something? I don't see any usage for this new field

Reply:
I did, but it was added with the last commit